File: Carthage/Checkouts/SwiftTheme/OCDemo/ListViewController.m
//
//  ListViewController.m
//  SwiftTheme
//
//  Created by Gesen on 16/5/26.
//  Copyright © 2016年 Gesen. All rights reserved.
//

#import "ListViewController.h"
#import "Global.h"
#import "OCDemo-Swift.h"

@interface ListViewController ()

@end

@implementation ListViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    self.tableView.theme_backgroundColor = globalBackgroundColorPicker;
    self.tableView.theme_separatorColor = [ThemeColorPicker pickerWithColors:@[@"#C6C5C5", @"#C6C5C5", @"#C6C5C5", @"#ECF0F1"]];
}

@end

File: Carthage/Checkouts/SwiftTheme/Demo/ListViewController.swift
//
//  ListViewController.swift
//  Demo
//
//  Created by Gesen on 16/3/1.
//  Copyright © 2016年 Gesen. All rights reserved.
//

import UIKit

class ListViewController: UITableViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        tableView.theme_backgroundColor = globalBackgroundColorPicker
        tableView.theme_separatorColor = ["#C6C5C5", "#C6C5C5", "#C6C5C5", "#ECF0F1"]
    }
    
}
